.github/workflows/labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: "Pull Request Labeler"
+on: [pull_request]
```

Comment:
As mentioned [here] this method won't work with `pull_request`. You should use something like [this] to schedule the workflow to run from time to time. Also I would add a workflow_dispatch just for testing purposes to check that this is working before merging it.

[here]: https://github.com/actions/labeler/issues/12
[this]: https://github.com/actions/starter-workflows/pull/78/files